src/Generator.Rewrite/AssemblyResolver.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Mono.Cecil;
+
+namespace OpenTK.Rewrite
+{
+    /// <summary>
+    /// A class that is responsible for resolving assembly references.<para/>
+    /// Note that this is currently exactly the same as the <see cref="BaseAssemblyResolver"/> because there is no need
+    /// for any additional functionality.
+    /// </summary>
+    public class AssemblyResolver : BaseAssemblyResolver
```

Comment:
If it's no longer needed, delete it.

Reply:
`BaseAssemblyResolver` is abstract, so this is still needed, even if empty.

Reply:
Can we simply not omit it? Cecil uses a `DefaultAssemblyResolver`, which inherits from BaseAssemblyResolver. All it does is add some caching.

Reply:
Oh, I completely missed the existence of that class. Yeah, will use that one instead.

Reply:
Iirc there was issue with default resolver

Reply:
Yes, there was an issue:
https://github.com/opentk/opentk/blob/3c6d7256293265d9bf6285bab8c5415d8ff5bae4/src/Generator.Rewrite/OpenTKAssemblyResolver.cs#L8-L12

This has been fixed though, the `BaseAssemblyResolver` now includes the exact same change.